src/web/index.php

```diff
@@ -92,14 +131,16 @@
                i.e. if you have filter_by_first_letter set you can additionally use filter_by_state
         -->
         <?php foreach ($airports as $airport): ?>
-        <tr>
-            <td><?= $airport['name'] ?></td>
-            <td><?= $airport['code'] ?></td>
-            <td><a href="#"><?= $airport['state'] ?></a></td>
-            <td><?= $airport['city'] ?></td>
-            <td><?= $airport['address'] ?></td>
-            <td><?= $airport['timezone'] ?></td>
-        </tr>
+            <tr>
+                <td><?= $airport['name'] ?></td>
+                <td><?= $airport['code'] ?></td>
+                <td>
+                    <a href="<?= generateUrl($request, 'filter_by_state', $airport['state'], true) ?>"><?= $airport['state'] ?></a>
```

Comment:
please use <?php echo instead of <?=

Reply:
OK.  I have updated code. I have a question, why we shouldn't use short tags? Is it related to different setting of config files?

Updated.